src/v2/Apps/Show/components/ShowArtworks.tsx

```diff
@@ -0,0 +1,152 @@
+import { ShowArtworks_show } from "v2/__generated__/ShowArtworks_show.graphql"
+import { BaseArtworkFilter } from "v2/Components/v2/ArtworkFilter"
+import { ArtworkFilterContextProvider } from "v2/Components/v2/ArtworkFilter/ArtworkFilterContext"
+import { updateUrl } from "v2/Components/v2/ArtworkFilter/Utils/urlBuilder"
+import React from "react"
+import { RelayRefetchProp, createRefetchContainer, graphql } from "react-relay"
+import { MediumFilter } from "v2/Components/v2/ArtworkFilter/ArtworkFilters/MediumFilter"
+import { PriceRangeFilter } from "v2/Components/v2/ArtworkFilter/ArtworkFilters/PriceRangeFilter"
+import { WaysToBuyFilter } from "v2/Components/v2/ArtworkFilter/ArtworkFilters/WaysToBuyFilter"
+import { SizeFilter } from "v2/Components/v2/ArtworkFilter/ArtworkFilters/SizeFilter"
+import { TimePeriodFilter } from "v2/Components/v2/ArtworkFilter/ArtworkFilters/TimePeriodFilter"
+import { ColorFilter } from "v2/Components/v2/ArtworkFilter/ArtworkFilters/ColorFilter"
+import { Box } from "@artsy/palette"
+import { useRouter } from "v2/Artsy/Router/useRouter"
+
+interface ShowArtworksFilterProps {
+  show: ShowArtworks_show
+  relay: RelayRefetchProp
+}
+
+const ShowArtworksFilter: React.FC<ShowArtworksFilterProps> = props => {
+  const { relay, show } = props
+  const { match } = useRouter()
+  const { filtered_artworks } = show
+
+  const hasFilter = filtered_artworks && filtered_artworks.id
+
+  if (!hasFilter) return null
+
+  const Filters = () => (
```

Comment:
It's nice to be able to specify `Filters` like this. Since this is a show page, we don't need to include the typical gallery and institution options.